### ISSUE TITLE
标签显示中文

### ISSUE THREAD

**Issue by zhangjingling** [valencebond/Strong_Baseline_of_Pedestrian_Attribute_Recognition#23]
您好，您知道如何把标签显示成中文吗？rap数据集有attribute_chinese字段，不知道怎麽用？

**Comment by zhangjingling** [comment]
# show the score in command line
            for idx in range(len(attr_list_rap)):
                if logits[idx]>=0:
                    print(attr_list_ch_rap[idx]+":",end="")
                    print(logits[idx])
                    #print('%s: %.8f'%(attr_list_ch_rap[idx].encode('utf-8').decode("latin1"), logits[idx]))
            

            # show the score in the image
            img = img.resize(size=(256, 512), resample=Image.BILINEAR)
            draw = ImageDraw.Draw(img)
            positive_cnt = 0
            font = ImageFont.truetype("simsun.ttc", 14, encoding="unic")
            for idx in range(len(attr_list_rap)):
                if logits[idx]>=0:
                   
                   txt = '%s: %.2f'%(attr_list_ch_rap[idx].encode('utf-8').decode('utf-8'), logits[idx])
                   draw.text((10, 10 + 20*positive_cnt), txt, (255,255, 0), font)
                   positive_cnt += 1
            img.save("./data/demo_result/" + file)``
